topics/icrc_27_get_icrc_1_accounts.md

```diff
@@ -0,0 +1,204 @@
+# ICRC-27: Get ICRC-1 Accounts
```

Comment:
This PR only adds a file. You should also delete the other existing file `icrc_27_get_icrc_1_subbaccounts.md`.

Reply:
Fixed, PR now correctly shows file as renamed.